configs/mushra_customquestionaire.yaml

```diff
@@ -29,7 +29,7 @@ pages:
     - type: finish
       name: Thank you
       content: Thank you for attending
-      popupcontent: Your results were sent. Goodbye and have a nice day
+      popupContent: Your results were sent. Goodbye and have a nice day
```

Comment:
I didn't see more changes elsewhere regarding this renaming. Is this fixing something?

Reply:
Yes, the option should be spelled with a capital C (as is done in other example config files). If spelled with lowercase, the option doesn't actually get set.